src/lvl_script.c

```diff
@@ -2921,6 +2922,25 @@ void command_change_creature_owner(long origin_plyr_idx, const char *crtr_name,
   command_add_value(Cmd_CHANGE_CREATURE_OWNER, origin_plyr_idx, crtr_id, select_id, dest_plyr_idx);
 }
 
+
+void command_computer_dig_to_location(long plr_range_id, const char* origin, const char* destination)
+{
+    TbMapLocation orig_loc;
+    if (!get_map_location_id(origin, &orig_loc))
+    {
+        SCRPTWRNLOG("Dig to location script command has invalid location: %s", origin);
+        return;
+    }
+    TbMapLocation dest_loc;
+    if (!get_map_location_id(destination, &dest_loc))
+    {
+        SCRPTWRNLOG("Dig to location script command has invalid location: %s", destination);
```

Comment:
Let message be different ie "source" in first case and "destionation" in second one